rmf_site_format/src/sdf.rs

```diff
@@ -18,18 +18,18 @@
 use crate::{
     Anchor, Angle, AssetSource, Category, DoorType, Level, LiftCabin, Pose, Rotation, Site, Swing,
 };
+use bevy::{prelude::*, utils::tracing};
```

Comment:
Why was `bevy` introduced here? Was that done to bring in `error!` from `tracing`?

I don't think anything from `bevy` is being used anymore, so we can remove this. It's currently causing compilation errors because we want `rmf_site_format` to be able to build without `bevy`.

Reply:
FYI to reproduce the build error locally you should be able to `cd` into the `rmf_site_format` directory and run `cargo test`.